### PR TITLE
Add landcover 2001 2020

### PR DESCRIPTION
**Pull request by vlulla**
Addresses #226

### CODE REVIEW

File: data/collections/bangladesh-landcover-2001-2020.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        "temporal": {
+            "interval": [
+                [
+                    "2001-01-01T00:00:00Z",
+                    "2020-12-31T23:59:59Z"
+                ]
+            ]
+        }
+    },
+    "license": "CC0",
```
Comment (anayeaye) [inline data/collections/bangladesh-landcover-2001-2020.json:26]: Tiny change: If you change this id to CC0-1.0 community tools like the browser will link out to the SPDX record https://spdx.org/licenses/CC0-1.0.html

**Review by anayeaye — APPROVED** [review]
Approve with small change to license id in collection

File: data/collections/bangladesh-landcover-2001-2020.json (hunk 1 above)
Comment (vlulla) [reply data/collections/bangladesh-landcover-2001-2020.json]: Thank you! Done.

**Comment by vlulla** [comment]
Made changes so that the netlify preview link at https://github.com/NASA-IMPACT/delta-config/pull/134 works. Please ensure that you run the datetime summaries after ingesting the stac items.

**Review by anayeaye — APPROVED** [review]

**Pull request merged**